Screen-Sharing-Video-Chat /js/app.js

```diff
@@ -0,0 +1,123 @@
+/* global OT API_KEY TOKEN SESSION_ID SAMPLE_SERVER_BASE_URL */
+
+let apiKey;
+let sessionId;
+let token;
+let screenPublisher;
+let session;
+let publisher;
+let canPublish = false;
+// Handling all of our errors here by alerting them
+function handleError(error) {
+  if (error) {
+    alert(error.message);
+  }
+}
+
+
+function toggleScreen() {
+  const screenSharing = document.getElementById('screen-sharing');
+  // If the screen publisher is connected to the session, unpublish screen and re-publish camera footage
+  if (screenPublisher.session) {
+    session.unpublish(screenPublisher);
+    session.publish(publisher, handleError);
+    screenSharing.classList.remove('toggle-button-on');
+    screenSharing.classList.add('toggle-button-off');
+  // Else, if the camera publisher is connected to the session, unpublish camera footage and re-publish screen
+  } else if (publisher.session) {
+    session.unpublish(publisher);
+    if (canPublish) {
+      session.publish(screenPublisher, handleError);
+    }
+    screenSharing.classList.add('toggle-button-on');
+    screenSharing.classList.remove('toggle-button-off');
+  }
+}
+
+function preventDefaults() {
```

Comment:
We can remove this (since we won't need to reuse the publisher ... since we won't stop publishing the camera stream).